decentraland/unity-renderer
# test: complete flow for OnPointerResult

## What does this PR change?

This PR is just a test branch to test the whole flow of the OnPointerResult

## How to test the changes?

1.-Create a ECS7 scene,
2.- Add OnPointerDown to an entity
3.- Check that you receive an OnPointerResult on the same entity


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
